src/app/form/form.component.ts

```diff
@@ -353,12 +353,13 @@ export class MainFormComponent implements OnInit {
   }
   onSubmit() {
     this.calculateCapacity();
-    this.requestBody.shiftLength = this.shiftLength != "" ? this.shiftLength.split(',') : this.requestBody.shiftLength;
-    this.requestBody.lowerLimitFactor = this.upperUtilization != "" ? this.upperUtilization : this.requestBody.lowerLimitFactor;
-    this.requestBody.upperLimitFactor = this.utilization != "" ? this.utilization : this.requestBody.upperLimitFactor;
+    this.requestBody.shiftLength = this.shiftLength.split(",");
```

Comment:
@aparnaacco879 Are you sure these fields cannot be empty string - "" ?

Reply:
Ye, because we are sending default values in it.

Reply:
What if user cleans the values in UI?

Reply:
It will give validation error then and will not allow the user to enter the submit button as all the fields are required.

Reply:
![Screenshot from 2020-05-11 14-34-15](https://user-images.githubusercontent.com/59680514/81544386-14682080-9395-11ea-8386-ecd3bcf23277.png)